### ISSUE TITLE
Error with installation of numpyro

### ISSUE THREAD

**Issue by flyaflya** [flyaflya/causact#61]
Following installation instructions leads to:

```
> drawsDF = graph %>% dag_numpyro() ## see "sample:..."
Error in py_run_string_impl(code, local, convert) : 
  ModuleNotFoundError: No module named 'jax.linear_util'
```

**Comment by flyaflya** [comment]
working on fix now.

**Comment by flyaflya** [comment]
fixed with ff1e2750e381a1e8f57409b6c7bdab53fc47abfd
